mev-commit-cli.sh

```diff
@@ -265,10 +270,69 @@ start_bridge(){
         contract-deployer
 }
 
-stop_bridge(){
+stop_hyperlane(){
     AGENT_BASE_IMAGE=gcr.io/abacus-labs-dev/hyperlane-agent@sha256:854f92966eac6b49e5132e152cc58168ecdddc76c2d390e657b81bdaf1396af0 PUBLIC_SETTLEMENT_RPC_URL="$public_rpc_url" SETTLEMENT_RPC_URL="$rpc_url" docker compose -f "$BRIDGE_PATH/hyperlane/docker-compose.yml" --profile bridge down
 }
 
+start_local_l1() {
+    DD_KEY=nil docker compose --profile local_l1 -f "$GETH_POA_PATH/geth-poa/docker-compose.yml" up -d --build
+    sleep 10
```

Comment:
It's because we wait for the settlement layer to come up before we run Deploy_create2. We might want some way to poll for a health check before proceeding instead if you think that's a better option.